src/libopensc/opensc.h

```diff
@@ -178,8 +201,11 @@ typedef struct sc_security_env {
 	struct sc_path file_ref;
 	unsigned char key_ref[8];
 	size_t key_ref_len;
+	struct sc_path target_file_ref; /* target key file in unwrap operation */
```

Comment:
No, because we need file_ref to identify the file where the unwrapping key (which performs the crypto-operation) resides. File pointed by target_file_ref receives the result of the operation (the unwrapped key data).